### PR TITLE
Point to existing tag

### PR DESCRIPTION
**Pull request by SashaWeinstein**
This PR is related to [this pluto upgrade](https://github.com/NYCPlanning/db-pluto/issues/386). I want to add a new tag to our [library image](https://hub.docker.com/r/nycplanning/library/tags) and I want this action to still work if I break something and can't fix it

### CODE REVIEW

**Comment by SashaWeinstein** [comment]
This is how to publish to this change after this is approved https://docs.github.com/en/actions/creating-actions/publishing-actions-in-github-marketplace

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM nycplanning/library:ubuntu-latest
+FROM nycplanning/library:ubuntu-5f514f801976479945dfbdc3fc0b9ec864e2e810
```
Comment (mbh329) [inline Dockerfile:1]: This is just a specific version of the docker image that you want to hold consistent?

Comment (mbh329) [reply Dockerfile:1]: This i my fault for not looking at this sooner but its unclear to me what the impact of this is/will be and how this changes our workflow

Comment (SashaWeinstein) [reply Dockerfile:1]: Yes this points to the working docker image that we've been using in our publishing workflows

Comment (SashaWeinstein) [reply Dockerfile:1]: Hopefully as we talk through these pull requests it becomes more clear

Comment (td928) [reply Dockerfile:1]: my understanding is this is a hack to not break anything that relies on the latest tag?

Comment (td928) [reply Dockerfile:1]: no it is way to revert back to the last working thing. got it now.

**Review by mbh329 — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (td928) [reply Dockerfile:1]: but I guess we also can't test if this tag will work until this merges in or if there are some other ways we can test this?

**Review by td928 — CHANGES_REQUESTED** [review]
holding approval to get more clarification

**Comment by SashaWeinstein** [comment]
Max in this situation if you don't feel you understand the potential impacts then it's best not to approve, fine to wait until we've talked it through

**Comment by mbh329** [comment]
The way I tested this was to see if the image was available on dockerhub https://registry.hub.docker.com/r/nycplanning/library/tags but yeah should have gotten some further clarification from you before approving

**Review by td928 — APPROVED** [review]

**Comment by SashaWeinstein** [comment]
This was all for naught as the new image I pushed didn't overwrite the `ubuntu-latest`. So we can keep v1.1, it still does what it did before.

**Pull request closed without merging**